src/main/java/persistence/sql/ddl/QueryBuilder.java

```diff
@@ -0,0 +1,59 @@
+package persistence.sql.ddl;
+
+import persistence.sql.validator.EntityValidator;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+import static persistence.sql.common.SqlConstant.*;
+
+public class QueryBuilder {
+    private final StringBuilder queryBuilder;
+    private final Field[] fields;
+    private final String tableName;
+
+    public QueryBuilder(Class<?> entity) {
+        EntityValidator.validate(entity);
+        this.fields = entity.getDeclaredFields();
+        this.tableName = new Table(entity).getName();
+        this.queryBuilder = new StringBuilder(String.format(CREATE_TABLE_START, tableName));
+    }
+
+    public String getCreateQuery() {
+        initQueryBuilder();
+        Arrays.stream(fields).forEach(field -> {
+            queryBuilder.append(new AnnotationFreeColumn(field).getColumn());
+            queryBuilder.append(CRETE_TABLE_COMMA);
+        });
+        endQuery();
+
+        return queryBuilder.toString();
+    }
```

Comment:
코드 각라인의 순서를 지켜져야지만 쿼리가 정상적으로 만들어지는 구조로 개발해주셨어요~
우리 좀 더 이해하기 쉽도록 수정해볼까요? 

```suggestion
    public String getCreateQuery() {
        return String.format(CREATE_TABLE_START, tableName) +
                getColumnQuery() +
                CREATE_TABLE_END;
    }

    private String getColumnQuery() {
        return Arrays.stream(fields).map(field -> {
                    return new AnnotationFreeColumn(field).getColumn();
                })
                .reduce((s1, s2) -> s1 + CRETE_TABLE_COMMA + s2)
                .orElse("");
    }
```

어떤가요? 조금은 직관적으로 변한게 느껴지셨나요?
정말 만들고자 하는것이 무엇인지 본질만 남겨질때까지 한번 리팩토링 해보자구요~

Reply:
반영했습니다!